### PR TITLE
Added iFrame quickstart along with links in concepts page

### PR DESCRIPTION
**Pull request by iampearceman**

### CODE REVIEW

File: quickstarts/iFrame.mdx [hunk 1]
```diff
@@ -0,0 +1,297 @@
+---
+title: 'iFrame'
+description: ''
```
Comment (unicodeveloper) [inline quickstarts/iFrame.mdx:3]: Please add a description. E.g `Learn how to use Novu to receive In-App notifications with iFrame embed.`

File: quickstarts/iFrame.mdx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+We're gonna use custom variables here
+
+Feel free to add only text for now and rename the workflow to quickstart. It automatically creates a slug-like Identifier that will be needed in later steps to trigger a notification.
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/4.png"
+/>
+
+Renaming the workflow to 'quickstart'
```
Comment (unicodeveloper) [inline quickstarts/iFrame.mdx:86]: Is this meant to be a header or a comment that refers to the image above?

File: getting-started/concepts.mdx [hunk 3]
```diff
@@ -6,54 +6,54 @@ icon: 'building-columns'
 
 ## Notification
 
-A notification conveys information from source to recipient, triggered by a workflow acting as a message blueprint. Notifications can be individual or bundled as digest for user-friendliness.
+A notification conveys information from source to recipient, triggered by a workflow acting as a message blueprint. Notifications can be individual or bundled as digest for user-friendliness.
```
Comment (unicodeveloper) [inline getting-started/concepts.mdx:9]: Link the digest word in this sentence to a part of the docs that explains digest in full.

File: quickstarts/iFrame.mdx (hunk 1 above)
Comment (iampearceman) [reply quickstarts/iFrame.mdx]: Done.

File: quickstarts/iFrame.mdx [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+Configure the in-app node as per your need
+
+I’ll briefly explain the function of each label in the image above:
+
+- **1-Preview**: This shows you a glimpse of how each notification item will look in the Notification Center UI.
+- **2-Avatar**: If turned on, each notification item will show the avatar of the subscriber.
+- **3-Action**: With this, you can add a primary and secondary call to action button to each notification item.
+- **4-Notification Feeds**: This displays a stream of specific notifications. You can have multiple feeds to show specific notifications in multiple tabs.
+- **5-Redirect URL**: This is the URL to which a subscriber can be directed when they click on a notification item.
+- **6-Filter**: This feature allows you to configure the criteria for delivering notifications. For instance, you can apply a filter based on a subscriber’s online status to send them an email if they were online within the last hour. Read **[more about filters](https://docs.novu.co/platform/step-filter/#subscriber-seen--read-filters)**.
```
[the remark this answers is not in the document]
Comment (iampearceman) [reply quickstarts/iFrame.mdx]: Done.

File: quickstarts/iFrame.mdx (hunk 2 above)
Comment (iampearceman) [reply quickstarts/iFrame.mdx]: Comment that refers to the image above. 

Done.

File: quickstarts/iFrame.mdx [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+## Create a subscriber
+
+If you click “Subscriber” on the left sidebar of the **[Novu dashboard](https://web.novu.co/subscribers)**, you’ll see the subscriber list. By default, there will only be one subscriber as you’re automatically added as a subscriber when you sign up for Novu:
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/5.png"
+/>
+
+One subscriber is automatically created when you sign up
```
[the remark this answers is not in the document]
Comment (iampearceman) [reply quickstarts/iFrame.mdx]: It's a note. Done.

File: quickstarts/iFrame.mdx [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+Select the in-app channel
+
+1. Click on the recently added “In-App” channel and configure it according to your preferences. Once you’re done, click “Update” to save your configuration.
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/2.png"
+/>
+
+Configure the in-app node as per your need
```
[the remark this answers is not in the document]
Comment (iampearceman) [reply quickstarts/iFrame.mdx]: Done.

File: getting-started/concepts.mdx (hunk 3 above)
Comment (iampearceman) [reply getting-started/concepts.mdx]: Done.

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**